src/App.jsx

```diff
@@ -14,70 +16,85 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [selected, setSelected] = useState(false);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood !== '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+          <button
+            onClick={() => {
+              setSelectedGood('');
+              setSelected(false);
+            }}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={selectedGood === good
+                ? 'has-background-success-light'
+                : undefined}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  onClick={() => {
+                    if (selected === false && selectedGood !== good) {
+                      setSelectedGood(good);
+                      setSelected(true);
+                    }
```

Comment:
create handler for click and try to optimize your solution, because 4 ifs is too much for this action